### PR TITLE
feat: Add CLI `update-oracle-info`

### LINKED ISSUE #9: Add CLI: update-oracle-info
request Tx to Panacea for updating oracle info

### PR DESCRIPTION
**Pull request by inchori**
Closes #9

### CODE REVIEW

File: cmd/oracled/cmd/update_oracle_info.go [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+				return err
+			}
+
+			// get trusted block information
+			trustedBlockInfo, err := getTrustedBlockInfo(cmd)
+			if err != nil {
+				return fmt.Errorf("failed to get trusted block info: %w", err)
+			}
+
+			queryClient, err := panacea.NewVerifiedQueryClient(context.Background(), conf, *trustedBlockInfo)
```
Comment (H4NLee) [inline cmd/oracled/cmd/update_oracle_info.go:31]: There is no need to receive trusted block info, actually must not.
Let's load verified query client using conf.
```go
queryClient, err := panacea.LoadVerifiedQueryClient(context.Background(), conf)
```

File: cmd/oracled/cmd/update_oracle_info.go [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+				return fmt.Errorf("update oracle info transaction failed: %v", resp.TxResponse.RawLog)
+			}
+
+			log.Infof("update-oracle-info transaction succeed. height(%v), hash(%s)", resp.TxResponse.Height, resp.TxResponse.TxHash)
+			return nil
+		},
+	}
+
+	cmd.Flags().Int64(flags.FlagTrustedBlockHeight, 0, "Trusted block height")
+	cmd.Flags().String(flags.FlagTrustedBlockHash, "", "Trusted block hash")
```
Comment (H4NLee) [inline cmd/oracled/cmd/update_oracle_info.go:91]:
```suggestion
```

File: cmd/oracled/cmd/update_oracle_info.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+	cmd.Flags().String(flagOracleEndpoint, "", "endpoint of oracle")
+	cmd.Flags().String(flagOracleCommissionRate, "", "oracle commission rate")
+	_ = cmd.MarkFlagRequired(flags.FlagTrustedBlockHeight)
+	_ = cmd.MarkFlagRequired(flags.FlagTrustedBlockHash)
```
Comment (H4NLee) [inline cmd/oracled/cmd/update_oracle_info.go:95]:
```suggestion
```

Comment (inchori) [reply cmd/oracled/cmd/update_oracle_info.go]: Fixed in [e3477c4](https://github.com/medibloc/panacea-oracle/pull/25/commits/e3477c482ab0e1288339e52757110655b1871931).

File: cmd/oracled/cmd/update_oracle_info.go (hunk 1 above)
Comment (inchori) [reply cmd/oracled/cmd/update_oracle_info.go]: Actually, I missed it 😂 Thx!

**Review by H4NLee — APPROVED** [review]
lgtm

**Review by audtlr24 — APPROVED** [review]
Good, I left one comment.

File: cmd/oracled/cmd/update_oracle_info.go [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+				return err
+			}
+
+			oracleCommissionRate, err := sdk.NewDecFromStr(oracleCommissionRateStr)
+			if err != nil {
+				return err
+			}
+
+			//TODO: The argument of NewMsgUpdateOracleInfo will be changed when https://github.com/medibloc/panacea-core/pull/540 is merged.
+			msgUpdateOracleInfo := oracletypes.NewMsgUpdateOracleInfo(oracleAccount.GetAddress(), oracleEndPoint, oracleCommissionRate)
```
[the remark this answers is not in the document]
Comment (inchori) [reply cmd/oracled/cmd/update_oracle_info.go:52]: Yes, when the `UpdateOracleInfo` Tx PR which is located in panacea-core is merged, I'll refer to it. Thank you!

**Pull request merged**